app/flows/am_i_getting_minimum_wage_flow/questions/how_old_are_you.erb

```diff
@@ -9,7 +9,3 @@
 <% text_for :error_message do %>
   Please enter a whole number greater than 0
 <% end %>
-
```

Comment:
Very minor feedback and not a blocker to merging: I would have split this out into a separate commit to be certain it doesn't affect the rest of the changes which feel unrelated.